cmd/openshift-install/testdata/imagebased/configimage/manifests/extra_manifests.txt

```diff
@@ -0,0 +1,63 @@
+# Verify that the extra-manifests directory populates with the appropriate files
+
+exec openshift-install image-based create config-image --dir $WORK
+
+exists $WORK/imagebasedconfig.iso
+
+isoContains imagebasedconfig.iso /extra-manifests/test.yaml
+isoContains imagebasedconfig.iso /extra-manifests/test.yml
+isoContains imagebasedconfig.iso /extra-manifests/test.json
+
+-- install-config.yaml --
+apiVersion: v1
+baseDomain: test.metalkube.org
+controlPlane:
+  name: master
+  replicas: 1
+compute:
+- name: worker
+  replicas: 0
+metadata:
+  name: ostest
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OVNKubernetes
+  machineNetwork:
+  - cidr: 192.168.111.0/24
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+sshKey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDK6UTEydcEKzuNdPaofn8Z2DwgHqdcionLZBiPf/zIRNco++etLsat7Avv7yt04DINQd5zjxIFgG8jblaUB5E5C9ClUcMwb52GO0ay2Y9v1uBv1a4WhI3peKktAzYNk0EBMQlJtXPjRMrC9ylBPh+DsBHMu+KmDnfk7PIwyN4efC8k5kSRuPWoNdme1rz2+umU8FSmaWTHIajrbspf4GQbsntA5kuKEtDbfoNCU97o2KrRnUbeg3a8hwSjfh3u6MhlnGcg5K2Ij+zivEsWGCLKYUtE1ErqwfIzwWmJ6jnV66XCQGHf4Q1iIxqF7s2a1q24cgG2Z/iDXfqXrCIfy4P7b/Ztak3bdT9jfAdVZtdO5/r7I+O5hYhF86ayFlDWzZWP/ByiSb+q4CQbfVgK3BMmiAv2MqLHdhesmD/SmIcoOWUF6rFmRKZVFFpKpt5ATNTgUJ3JRowoXrrDruVXClUGRiCS6Zabd1rZ3VmTchaPJwtzQMdfIWISXj+Ig+C4UK0=
+pullSecret: '{"auths": {"quay.io": {"auth": "c3VwZXItc2VjcmV0Cg=="}}}'
+
+-- image-based-config.yaml --
+apiVersion: v1beta1
+kind: ImageBasedConfig
+metadata:
+  name: ostest
+  namespace: cluster0
+hostname: ostest
+
+-- extra-manifests/test.yaml --
+apiVersion: v1beta1
+kind: DummyData
+metadata:
+  name: test.yaml
+
+-- extra-manifests/test.yml --
+apiVersion: v1beta1
+kind: DummyData
+metadata:
+  name: test.yml
```

Comment:
one of these should be `another-test.yml`, no?

Reply:
I think `test.yml` is enough, as this here is the input file to the installer (and `metadata.name` contains the suffix). In the end, we have one manifests for each pattern, `.yaml`, `.yml`, `.json`, were you thinking about an additional test case (that I'm missing)?

Reply:
Oh but this here is for the intergation tests, there is no connection to the unit tests. There is no technical need for those files to have the same names.